### ISSUE TITLE
Wording of `unnecessary_null_comparison`

### ISSUE THREAD

**Issue by Levi-Lesches** [dart-lang/linter#4796]
**Describe the issue**
The `unnecessary_null_comparison` lint has an incorrect message.

**To Reproduce**
```dart
void main() {
  int? something;
  if (something == null) {  // Warning: The operand can't be null, so the condition is always 'true'.
    print("This always prints");
  }
}
```

**Expected behavior**
I believe the correct message should be: `The operand is always null, so the condition is always 'true'.`

Also, the "fix" message is `Remove the condition.` which may be a bit harsh compared to the other messages. How about `Try removing the condition.`? Obviously not as important compared to the real issue.
